app/sagas/selectServer.js

```diff
@@ -145,6 +145,7 @@ const handleServerRequest = function* handleServerRequest({ server, certificate
 			yield put(selectServerRequest(server, serverInfo.version, false));
 		}
 	} catch (e) {
+		showErrorAlert(I18n.t('Cant_get_public_settings', { contact: I18n.t('Contact_your_server_admin') }), I18n.t('Oops'));
```

Comment:
If `getServerInfo` (or something else) throws an error, it's going to raise an alert with `Can't get public settings`, which may cause confusion.